bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/handler/YamahaBridgeHandler.java

```diff
@@ -167,9 +167,8 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                     systemControl.setPartyModeMute(((OnOffType) command) == OnOffType.ON);
                     break;
                 case CHANNEL_PARTY_MODE_VOLUME:
-                    if (command instanceof IncreaseDecreaseType) {
-                        systemControl
-                                .setPartyModeVolume(((IncreaseDecreaseType) command) == IncreaseDecreaseType.INCREASE);
+                    if (command instanceof IncreaseDecreaseType castedCommand) {
+                        systemControl.setPartyModeVolume(castedCommand == IncreaseDecreaseType.INCREASE);
```

Comment:
Perhaps slightly more descriptive:
```suggestion
                    if (command instanceof IncreaseDecreaseType increaseDecreaseCommand) {
                        systemControl.setPartyModeVolume(increaseDecreaseCommand == IncreaseDecreaseType.INCREASE);
```

Reply:
No, you're right. This is better.